paper/paper.bib

```diff
@@ -0,0 +1,11 @@
+@article{Scheidgen:2023,
+  doi = {10.21105/joss.05388},
+  url = {https://doi.org/10.21105/joss.05388},
+  year = {2023},
+  volume = {8},
+  number = {90},
+  pages = {5388},
+  author = {Markus Scheidgen and Lauri Himanen and Alvin Noe Ladines and David Sikter and Mohammad Nakhaee and {Ádám Fekete} and Theodore Chang and Amir Golparvar and {José A. Márquez} and Sandor Brockhauser and Sebastian Br{ü}ckner and Luca M. Ghiringhelli and Felix Dietrich and Daniel Lehmberg and Thea Denell and Andrea Albino and Hampus N{ä}sstr{ö}m and Sherjeel Shabih and Florian Dobener and Markus K{ü}hbach and Rubel Mozumder and Joseph F. Rudzinski and Nathan Daelman and {José M. Pizarro} and Martin Kuban and Cuauhtemoc Salazar and Pavel Ondra{č}ka and Hans-Joachim Bungartz and Claudia Draxl},
```

Comment:
We should check that special characters, like in Adam's, Pavel's case and German umlauts get compiled correctly.